src/components/typography/text/text.styles.js

```diff
@@ -48,15 +48,15 @@ const getTone = tone => {
 const getElementFontSize = elementType => {
   switch (elementType) {
     case 'h1':
-      return '32px';
+      return '2.462rem';
```

Comment:
used https://offroadcode.com/rem-calculator/

Reply:
Can you even calc the thing maybe? Like calc(var(—root-font-size * x))em? I don‘t even know. Was just wondering.

Reply:
I wouldn't put --root-font-size here though, because the whole advantage of using REM is that a user could use a different base size... 

Instead maybe we could decide that h1 should be X percentage bigger than 1, and use that for the calculation?

Reply:
For example, if I go to the netlify branch deploy and I change the root font size to 16, then the computed value for our H1 becomes 39.3846px, while at the current root font size of 13 it's ~32px.

Reply:
Yeah I guess we could have something like 

`--h1-scale` = 32 / 13 = `2.46153846154`

Reply:
That's what I meant, cause the numbers are otherwise kind of hard to reconstruct or understand and hence to adjust. Given we just have the scales it also quickly shows if they actually make sense or if they're kind of random (not that we should change that on this PR).